jobclient/src/main/java/com/twosigma/cook/jobclient/constraint/Constraints.java

```diff
@@ -66,4 +66,26 @@ public static Constraint parseFrom(JSONArray constraint) {
                 throw new UnsupportedOperationException(operator + " is not supported.");
         }
     }
+
+    /**
+     * @param constraint the string representation of a constraint. The string could be either the form
+     *                   "attribute,operator" or the form "attribute,operator,value1,value2,...".
+     *                   See {@link Constraint}.
+     * @return the parsed constraint.
+     */
+    public static Constraint parseFrom(String constraint) {
```

Comment:
What caused you to want to add this?

Reply:
See this https://github.com/twosigma/spark/pull/27/files

Reply:
I would like to make sure you are happy to add this API first. Then I will do some refactoring later